File: loop/1067.py
#1067 NÚMEROS ÍMPARES
#1° Solução com for
x = int(input())

for i in range(x+1):
    if i%2 != 0:
        print(i)
        i += 1

#2° Solução com while
"""
x = int(input())
i = 1

while i <= x:
    if i%2 != 0:
        print(i)
    i += 1
"""

File: javascript/1067.js
/*Números Ímpares*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split("\n");

x = parseInt(lines.shift());

i = 1;
while(i <= x){
  if(i % 2 !== 0)
   console.log(i);
   i++;
}